### ISSUE TITLE
custom dataset very high vall and labelling problems

### ISSUE THREAD

**Issue by msts35** [dusty-nv/pytorch-ssd#1]
Hi, as you suggested i reproduce the repository and try to use it,

when i try to train ssd mobilenet with my labelled dataset it firstly gives errors about the CSV format that i had created before via xml_to_csv.py script of tensorflow object detection api, i changed some row IDs to fit open-image dataset type and it can start to be trained

however the loss values are very high, they are over 5k and did not decrease, i think the problem is about labels because i can train guns dataset with acceptable values 

what can you suggest? can you please give me a advice about labeling images that can suitable ? my dataset has small rect boxes, does it make problems?

thank you for the great repository and everything that you provide
